versions.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc.
+# Copyright 2019 Google LLC
```

Comment:
why not 2020?

Reply:
nice point. the rotationlib should be updated. :+1: